app/models/team.rb

```diff
@@ -45,18 +45,23 @@ def visible
     end
 
     # Returns a base scope reflecting the relevant teams depending on what
-    # phase of the running season we're currently in.
+    # phase of the running season we're currentin_ly in.
     def by_season_phase
       if Time.now.utc > Season.current.acceptance_notification_at
-        Team.where season: Season.current, kind: %w(sponsored voluntary)
+        Team.in_current_season.selected
       else
-        Team.visible.where season: Season.current
+        Team.in_current_season.visible
       end
     end
 
-    def active_teams
-      Team.where(season: Season.current, kind: %w(sponsored voluntary)).order(:name)
+    def in_current_season
```

Comment:
There are traits that are also referenced by `:current`. They should be called `:in_current_season` as well. But let's discuss the details in the new PR.

As for the merge: see my most recent comment below. If that's done, I'll merge in an instant :)